src/howto_private_tx.md

```diff
@@ -16,18 +16,19 @@ Build worker, client and node in our docker:
 
 ```bash
 # get the docker image
-# check for updates on https://hub.docker.com/repository/docker/scssubstratee/substratee_dev
-docker pull scssubstratee/substratee_dev:1804-2.12-1.1.3-001
+# check for updates on https://hub.docker.com/r/integritee/integritee-dev/tags
+docker pull integritee/integritee-dev:0.1.7
 
 # create a dedicated demo directory and start the docker container
 mkdir demo && cd demo
-docker run -it -v $(pwd):/root/work scssubstratee/substratee_dev:1804-2.12-1.1.3-001 /bin/bash
+docker run -it -v $(pwd):/root/work integritee/integritee-dev:0.1.7 /bin/bash
 
 # now you are inside the container
 # clone and build the worker and the client
 cd work
 git clone https://github.com/integritee-network/worker.git
 cd worker
+rustup show
```

Comment:
Maybe you want to note here that this command installs to correct rust-toolchain to run the worker.

Reply:
Did it, thanks for the tip.

Reply:
Could you also add this to the how to _direct_tx demo?

Reply:
And with this the 

```
# initialize wasm build environment
./scripts/init.sh
```
below is obsolete now?

Reply:
yes, this is actually obsolete in most of our repos.

Reply:
Thanks for your feedback, changes are already in for your review again.